### ISSUE TITLE
Colors don't work in Windows 10 command prompt + powershell

### ISSUE THREAD

**Issue by SemmieDev** [dialex/JCDP#54]
**Describe the bug**
This works fine everywhere else for me except in the command prompt and powershell in Windows 10. I am not talking about the Windows Terminal app, it works fine in there.

**To Reproduce**
1. Make a new Java project or edit an existing one.
2. Add a simple line of code that will log colorized text into console.
3. Run the project in the command prompt or in powershell.
4. The colorized text will have gibberish around it.

**Expected behavior**
I expected the text to be colorized, which I have seen some programs doing.

**Screenshots**
Powershell:
![image](https://user-images.githubusercontent.com/58337334/108971679-d9fd1500-7682-11eb-85a9-7f8a220f61b0.png)
Command prompt:
![image](https://user-images.githubusercontent.com/58337334/108971766-f00ad580-7682-11eb-9a26-5ad595a7d2d8.png)
Windows Terminal (expected behavior):
![image](https://user-images.githubusercontent.com/58337334/108971914-1892cf80-7683-11eb-93fa-2779b64d2314.png)

**Environment**
 - OS: Windows
 - Version: 10

**Comment by dialex** [comment]
Apparently this is the reason why it doesn't work for cmd and powershell: https://superuser.com/a/1300251/97034

> the new support for ANSI Terminal Control which was added to the Windows 10 console host in build 16257 (and later). Namely, it's n̲o̲t̲ e̲n̲a̲b̲l̲e̲d̲ by default. Unless the specific software you're using enables ANSI processing by calling the SetConsoleMode API with the ENABLE_VIRTUAL_TERMINAL_PROCESSING (0x0400) flag, you won't see colors or get ANSI processing for that application.

It's confirmed again at https://stackoverflow.com/a/35864976/675577

I'll try to fix it with some kind of `if windows then set variable`. Apparently I used to have that behaviour https://github.com/dialex/JColor/commit/e4673339eea5c43f361ee5c707fbbfaa68011e08, through this method https://github.com/dialex/JColor/blob/e4673339eea5c43f361ee5c707fbbfaa68011e08/src/main/java/com/diogonunes/jcdp/color/ColoredPrinter.java#L339, but dropped it for some reason. I think it was because I thought after all this time Microsoft had enabled that behaviour by default.

P.S: Thanks for raising the ticket with all those details! 👌

**Comment by dialex** [comment]
Looks like you as the user can enable that flag once and forever: https://stackoverflow.com/a/51681675/675577

Can you try these instructions on your machine, and tell me if it fixes the issue? Don't run both, choose powershell or cmd, and run the fix only for that one. We want to have one of the programs behaving incorrectly so we can test my fix later.

----

* (a) **Activate support *globally by default, persistently***, via the registry, as detailed in [this SU answer](https://superuser.com/a/1300251/139307).
   * In short: In registry key `[HKEY_CURRENT_USER\Console]`, create or set the `VirtualTerminalLevel` DWORD value to `1` 
     * From PowerShell, you can do this *programmatically* as follows:  
`Set-ItemProperty HKCU:\Console VirtualTerminalLevel -Type DWORD 1`
     * From `cmd.exe` (also works from PowerShell):  
`reg add HKCU\Console /v VirtualTerminalLevel /t REG_DWORD /d 1`
   * Open a new console window for changes to take effect.
